lila/src/main/scala/Bus.scala

```diff
@@ -3,23 +3,36 @@ package bus
 
 import scala.jdk.CollectionConverters.*
 import scala.reflect.Typeable
+import scala.util.NotGiven
 import scala.concurrent.duration.*
 import scala.concurrent.{ ExecutionContext, Future, Promise }
+
+import typemap.{ MutableTypeMap, typeName, assertBuseable }
+
 import scalalib.future.extensions.withTimeout
 import scalalib.future.FutureAfter
 
 type Channel = String
 
+// BC
 // constructor is private so instances can only be created by extending the `GivenChannel` trait
 final class WithChannel[T](private val key: Channel):
   def channel: Channel = key
 
+// BC
 transparent trait GivenChannel[T](val channel: Channel):
   given WithChannel[T] = WithChannel[T](channel)
 
+// Marker trait to implement on types you specifically do not want to Bus
```

Comment:
should it be the other way around, like we only `bus` types that extends `Busable`?

Reply:
Yeah, I was wondering about that too. It's definitely safer to have a Busable trait. Would cause quite a bit of churn at introduction.